modules/heat_conduction/doc/content/source/fvbcs/FVThermalResistanceBC.md

```diff
@@ -0,0 +1,93 @@
+# FVThermalResistanceBC
+
+!syntax description /FVBCs/FVThermalResistanceBC
+
+# Description
+
+Heat flux boundary condition for the fluid or solid energy conservation equations. When used
+with the fluid energy equation, this boundary condition specifies
+
+\begin{equation}
+-\int_\Gamma\kappa_f\cdot\nabla T_f\cdot\hat{n}d\Gamma\ ,
+\end{equation}
+
+where
+
+\begin{equation}
+-\kappa_f\cdot\nabla T_f\cdot\hat{n}=\tilde{q}\ ,
+\end{equation}
+
+where $\tilde{q}$ is the imposed heat flux, to be discussed shortyl.
+When used with the solid energy equation, this boundary condition specifies
+
+\begin{equation}
+-\int_\Gamma\kappa_s\cdot\nabla T_s\cdot\hat{n}d\Gamma\ ,
+\end{equation}
+
+where
+
+\begin{equation}
+-\kappa_s\cdot\nabla T_s\cdot\hat{n}=\tilde{q}\ .
+\end{equation}
+
+The heat flux $\tilde{q}$ is computed based on the thermal resistance concept, where heat
+is assumed to conduct through multiple parallel slabs of constant-property solid in series,
+followed by parallel convection and radiation to an ambient temperature. For $N$ conducting
+layers, the conduction thermal resistance $R_c$ is computed for Cartesian slabs as
+
+\begin{equation}
+R_c=\sum_{i=1}^N\frac{\Delta x_i}{k_i}\ ,
+\end{equation}
+
+where $\Delta x_i$ is the thickness of each layer and $k_i$ is the thermal conductivity of
+each layer. For cylindrical annuli, the conduction thermal resistance is computed as
+
+\begin{equation}
+R_c=\sum_{i=1}^N\frac{\ln{\left(\frac{\Delta x_i+r_i}{r_i}\right)}}{k_i}\ ,
```

Comment:
Sooo... I pretty much just copied the finite element version of this object from Pronghorn including its documentation. The multiplication by 2pi is not currently happening in the code. Should it be?

Reply:
It's been a while since I wrote this, but I can say two things:

1. This class has been tested by comparing the temperature with a model that actually explicitly models the conduction heat transfer through 1-D layers. The tests show the correct behavior with the resistance BC.
2. The 2\pi shows up on traditional textbook definitions of thermal resistance, but IIRC, the reason it doesn't appear in the class or in this documentation is because it eventually would cancel with a 2\pi factor in the area somewhere else when computing the total heat flux.

So I suppose technically there should be a 2\pi in the documentation for the thermal resistance, but it's just not in the class implementation because it cancels anyways.

Reply:
Thanks for the clarification @aprilnovak. Given that, I will add it to the doc!

Reply:
Thanks for the careful read @GiudGiud!